tests/test_tokenization_fast.py

```diff
@@ -748,29 +746,37 @@ def assert_offsets_with_special_characters(self, tokenizer_r):
             add_special_tokens=True,
         )
 
+        do_lower_case = tokenizer_r.init_kwargs.get("do_lower_case")
```

Comment:
I don't think we test for both values of `do_lower_case`, but I kept doing it just in case we need it.

Reply:
Got it, don't hesitate to add tests if you feel like we need some more